pkg/ui/input.go

```diff
@@ -98,46 +98,47 @@ func (tui *TUI) initKeyboard() {
 			RemoveSlide(slideNum, tui)
 			tui.TerminalInputBuffer = []rune{}
 			return nil
-		} else if event.Key() == tcell.KeyBackspace || event.Key() == tcell.KeyBackspace2 {
-			if len(tui.TerminalInputBuffer) > 0 {
-				if CursorPos > 0 {
-					tui.TerminalInputBuffer = append(tui.TerminalInputBuffer[:CursorPos-1], tui.TerminalInputBuffer[CursorPos:]...)
-					CursorPos--
-				} else {
-					tui.TerminalInputBuffer = tui.TerminalInputBuffer[:len(tui.TerminalInputBuffer)-1]
-				}
-			}
-			if len(tui.TerminalLastChars) > len("exit") {
-				tui.TerminalLastChars = tui.TerminalLastChars[len(tui.TerminalLastChars)-len("exit"):]
-			}
-			// Working on the input buffer
-		} else if event.Key() == tcell.KeyRune {
-			if CursorPos >= len(tui.TerminalInputBuffer) {
-				// Append new rune to end of input buffer
-				tui.TerminalInputBuffer = append(tui.TerminalInputBuffer, event.Rune())
-				tui.TerminalLastChars = append(tui.TerminalLastChars, event.Rune())
-				if len(tui.TerminalLastChars) > len("exit") {
-					tui.TerminalLastChars = tui.TerminalLastChars[1:]
-				}
-			} else {
-				// Insert new rune at cursor position in input buffer
-				tui.TerminalInputBuffer = append(tui.TerminalInputBuffer[:CursorPos], append([]rune{event.Rune()}, tui.TerminalInputBuffer[CursorPos:]...)...)
-				tui.TerminalLastChars = append(tui.TerminalLastChars[:CursorPos], append([]rune{event.Rune()}, tui.TerminalLastChars[CursorPos:]...)...)
-				if len(tui.TerminalLastChars) > len("exit") {
-					tui.TerminalLastChars = tui.TerminalLastChars[1:]
-				}
-			}
-			CursorPos++
-
-		} else if event.Key() == tcell.KeyEnter {
-			if string(tui.TerminalInputBuffer) == "exit" || string(tui.TerminalLastChars) == "exit" {
-				slideNum, _ := strconv.Atoi(tui.TerminalPageBar.GetHighlights()[0])
-				RemoveSlide(slideNum, tui)
-			}
-			tui.TerminalInputBuffer = []rune{}
-			tui.TerminalLastChars = []rune{}
-			CursorPos = 0
 		}
+		// } else if event.Key() == tcell.KeyBackspace || event.Key() == tcell.KeyBackspace2 {
```

Comment:
Any reason why all these lines have been commented?

Reply:
Hey @supreeth7 
These have been commented for now
These are sort of the hacks Deepanshu had added for the exit case handling
I haven't gone through how much code would break if those are removed

I am working on the cleanup and removing unused code.  This will be removed/modified with those changes in the next PR